integration-cli/docker_api_containers_test.go

```diff
@@ -305,9 +311,9 @@ func (s *DockerAPISuite) TestGetContainerStatsNoStream(c *testing.T) {
 		assert.NilError(c, err)
 		defer apiClient.Close()
 
-		stats, err := apiClient.ContainerStats(testutil.GetContext(c), name, false)
+		stats, err := apiClient.ContainerStats(testutil.GetContext(c), name, client.ContainerStatsOptions{})
 		assert.NilError(c, err)
```

Comment:
Failure is here (`type with url containerd.runhcs.stats.v1.Statistics: not found`)


Coming from here; https://github.com/moby/moby/blob/0af6203b46ac8973fc0c0c7f2a95fa8c2e57132d/vendor/github.com/containerd/typeurl/v2/types.go#L278-L300